cmd/influxd/run/server_suite_test.go

```diff
@@ -19,13 +19,13 @@ func init() {
 			&Query{
 				name:    "create database should succeed",
 				command: `CREATE DATABASE db0`,
-				exp:     `{"results":[{}]}`,
+				exp:     `{"results":[{"statement_id":0}]}`,
```

Comment:
Yes, you would. But the point of this PR is to also give you information about what is being run interactively for chunking.

A different question. What benefit is there for statement ID being omitted from these results? It seems like an unnecessary complication to omit the statement id specifically for those results. Clients are supposed to check for error, not a blank JSON response. It looks like `influxdb-python` checks for the `error` field.